barretenberg/cpp/src/barretenberg/plonk_honk_shared/arithmetization/mega_arithmetization.hpp

```diff
@@ -309,13 +310,18 @@ template <typename FF_> class MegaArith {
             info("");
         }
 
-        size_t get_total_structured_size()
+        size_t get_structured_dyadic_size()
         {
-            size_t total_size = 0;
```

Comment:
I think this being 0 was just a bug that never showed up in any tests since +1 never made a difference in the dyadic size

Reply:
interesting. I guess we need more tests for borderline circuit sizes